lighthouse-core/gather/connections/connection.js

```diff
@@ -96,8 +96,8 @@ class Connection {
       this._callbacks.delete(object.id);
       if (object.error) {
         log.formatProtocol('method <= browser ERR',
-            {method: callback.method, params: object.result}, 'error');
-        callback.reject(object.result);
+            {method: callback.method}, 'error');
+        callback.reject(object.error);
```

Comment:
I like the idea of both. Make the source of the error clear in the message (at the very least it'll make bug reports easier to track down) and standardize on `reject(new Error(msg))` so that we can depend on the shape.